webpack.config.js

```diff
@@ -16,7 +16,8 @@ const CONFIG = loadConfig()
 const config = overrideEnvConfig(CONFIG)
 const TRADE_APP = { name: 'trade', title: 'Gnosis Protocol Exchange', filename: 'trade.html' }
 const EXPLORER_APP = { name: 'explorer', title: 'Gnosis Protocol Explorer', filename: 'index.html' }
-const ALL_APPS = [TRADE_APP, EXPLORER_APP]
+const SAFE_SWAP_APP = { name: 'safe-swap', title: 'Gnosis Safe Swap', filename: 'index.html' }
```

Comment:
The title feels like this is something from the Safe.
What about: 

```suggestion
const SAFE_SWAP_APP = { name: 'safe-swap', title: 'Gnosis Swap - Safe app', filename: 'index.html' }
```